c-bindings/BUILD.bazel

```diff
@@ -7,6 +7,7 @@ go_library(
         "internal.h",
         "json.cpp",
         "json.h",
+        "libgojsonnet.h",
```

Comment:
Please see [my comment from #281](https://github.com/google/go-jsonnet/pull/281#issuecomment-497971532) about this entry. I think we should leave it out here, though I still don't know of a way to keep it from coming back.

Reply:
Hmmm..., this way, it's only a matter of time until it slips back. I created a separate PR #308 which hopefully will keep it out.